### DISCUSSION TITLE
rsbuild+svelte+js  vscode cannot jump normally

### DISCUSSION THREAD

**Discussion by Liu7i-67** [web-infra-dev/rsbuild#1110] [Q&A] [answered]
I created the svelte+js project using the `pnpm create rsbuild@latest` command

I modified the configuration of rsbuild.config.mjs as follows:
```js
import { defineConfig } from "@rsbuild/core";
import { pluginSvelte } from "@rsbuild/plugin-svelte";

export default defineConfig({
  plugins: [pluginSvelte()],
  server: {
    port: "9001",
  },
  html: {
    template: "./index.html",
  },
  resolve: {
    extensionAlias: {
      ".svelte": ["/index.svelte", ".js", ".css", ".json"],
    },
  },
  source: {
    alias: (opts) => {
      opts["@"] = "./src";
      opts["pages"] = "./src/pages";
      opts["assets"] = "./src/assets";
      opts["components"] = "./src/components";
      opts["tools"] = "./src/tools";
    },
  },
});
```
There is no problem with the program running, but there is no reaction when I want to click on the red box to jump to the corresponding location in vscode.
![image](https://github.com/web-infra-dev/rsbuild/assets/95481822/5dff7bdd-f69b-47e2-96cf-c0e1d5d1e7c3)

What am I supposed to do, ask for help.

**Comment by chenjiahan** [comment, marked as answer]
You can configure aliases through the paths configuration in tsconfig.json, which is the recommended approach in TypeScript projects as it also resolves the TS type issues related to path aliases.

See: https://rsbuild.dev/guide/advanced/alias#using-tsconfigjsons-paths-configuration

**Reply by Liu7i-67** [reply]
Ok, thanks a lot. After I configured tsconfig.json, I can jump normally
